-------------------------------------------------------------------------------
Test set: implementaciones.ReservacionesBOTest_SinMocks
-------------------------------------------------------------------------------
Tests run: 11, Failures: 5, Errors: 0, Skipped: 0, Time elapsed: 1.051 s <<< FAILURE! -- in implementaciones.ReservacionesBOTest_SinMocks
implementaciones.ReservacionesBOTest_SinMocks.testEliminarReservacion_Exitoso -- Time elapsed: 0.009 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Unexpected exception thrown: excepciones.ServicioException: No se encontro la reservacion a eliminar
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:152)
	at org.junit.jupiter.api.AssertDoesNotThrow.createAssertionFailedError(AssertDoesNotThrow.java:84)
	at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:53)
	at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:36)
	at org.junit.jupiter.api.Assertions.assertDoesNotThrow(Assertions.java:3168)
	at implementaciones.ReservacionesBOTest_SinMocks.testEliminarReservacion_Exitoso(ReservacionesBOTest_SinMocks.java:168)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: excepciones.ServicioException: No se encontro la reservacion a eliminar
	at implementaciones.ReservacionesBO.eliminarReservacion(ReservacionesBO.java:137)
	at implementaciones.ReservacionesBOTest_SinMocks.lambda$testEliminarReservacion_Exitoso$5(ReservacionesBOTest_SinMocks.java:168)
	at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:49)
	... 6 more

implementaciones.ReservacionesBOTest_SinMocks.testCancelarReservacion_Exitoso -- Time elapsed: 0.002 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Unexpected exception thrown: excepciones.ServicioException: La reservacion ya fue cancelada con anterioridad
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:152)
	at org.junit.jupiter.api.AssertDoesNotThrow.createAssertionFailedError(AssertDoesNotThrow.java:84)
	at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:53)
	at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:36)
	at org.junit.jupiter.api.Assertions.assertDoesNotThrow(Assertions.java:3168)
	at implementaciones.ReservacionesBOTest_SinMocks.testCancelarReservacion_Exitoso(ReservacionesBOTest_SinMocks.java:69)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: excepciones.ServicioException: La reservacion ya fue cancelada con anterioridad
	at implementaciones.ReservacionesBO.cancelarReservacion(ReservacionesBO.java:68)
	at implementaciones.ReservacionesBOTest_SinMocks.lambda$testCancelarReservacion_Exitoso$1(ReservacionesBOTest_SinMocks.java:69)
	at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:49)
	... 6 more

implementaciones.ReservacionesBOTest_SinMocks.testObtenerReservacionesDeMesa_Exitoso -- Time elapsed: 0.004 s <<< FAILURE!
org.opentest4j.AssertionFailedError: La lista no debe estar vacia ==> expected: <false> but was: <true>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at org.junit.jupiter.api.AssertFalse.failNotFalse(AssertFalse.java:63)
	at org.junit.jupiter.api.AssertFalse.assertFalse(AssertFalse.java:36)
	at org.junit.jupiter.api.Assertions.assertFalse(Assertions.java:239)
	at implementaciones.ReservacionesBOTest_SinMocks.testObtenerReservacionesDeMesa_Exitoso(ReservacionesBOTest_SinMocks.java:48)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)

implementaciones.ReservacionesBOTest_SinMocks.testActualizarReservacion_Exitoso -- Time elapsed: 0.029 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Unexpected exception thrown: excepciones.ServicioException: La reservacion que se desea actualizar ya concluyo o fue cancelada
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:152)
	at org.junit.jupiter.api.AssertDoesNotThrow.createAssertionFailedError(AssertDoesNotThrow.java:84)
	at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:53)
	at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:36)
	at org.junit.jupiter.api.Assertions.assertDoesNotThrow(Assertions.java:3168)
	at implementaciones.ReservacionesBOTest_SinMocks.testActualizarReservacion_Exitoso(ReservacionesBOTest_SinMocks.java:124)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: excepciones.ServicioException: La reservacion que se desea actualizar ya concluyo o fue cancelada
	at implementaciones.ReservacionesBO.actualizarReservacion(ReservacionesBO.java:128)
	at implementaciones.ReservacionesBOTest_SinMocks.lambda$testActualizarReservacion_Exitoso$4(ReservacionesBOTest_SinMocks.java:124)
	at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:49)
	... 6 more

implementaciones.ReservacionesBOTest_SinMocks.testAgregarReservacion_Exitoso -- Time elapsed: 0.020 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Unexpected exception thrown: excepciones.ServicioException: No se puede realizar la reservacion debido a que el cliente cuenta con una reservacion activa
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:152)
	at org.junit.jupiter.api.AssertDoesNotThrow.createAssertionFailedError(AssertDoesNotThrow.java:84)
	at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:53)
	at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:36)
	at org.junit.jupiter.api.Assertions.assertDoesNotThrow(Assertions.java:3168)
	at implementaciones.ReservacionesBOTest_SinMocks.testAgregarReservacion_Exitoso(ReservacionesBOTest_SinMocks.java:107)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: excepciones.ServicioException: No se puede realizar la reservacion debido a que el cliente cuenta con una reservacion activa
	at implementaciones.ReservacionesBO.agregarReservacion(ReservacionesBO.java:118)
	at implementaciones.ReservacionesBOTest_SinMocks.lambda$testAgregarReservacion_Exitoso$3(ReservacionesBOTest_SinMocks.java:107)
	at org.junit.jupiter.api.AssertDoesNotThrow.assertDoesNotThrow(AssertDoesNotThrow.java:49)
	... 6 more

